adobe/react-spectrum
# Update cursor style to default for Tabs in docs

Closes https://github.com/adobe/react-spectrum/issues/5972

## ✅ Pull Request Checklist:


## 📝 Test Instructions:


1. Open the `useTabList` documentation page.
2. Hover over any tab in the examples.
3. Verify the cursor is now the default style, not text.

## 🧢 Your Project:
